src/stylesheets/objects/_content-layouts.scss

```diff
@@ -45,24 +45,25 @@
   margin: 0 auto;
   width: 100%;
 
-  @include media($desktop) {
-    width: $desktop;
+  @include media($medium) {
+    width: $medium;
   }
 
-  @include media($desktop-medium) {
+  @include media($large) {
     padding: 0 round($base-spacing-unit * 10);
-    width: $desktop-medium;
+    width: $large;
   }
 
-  @include media($desktop-large) {
-    width: $desktop-large;
+  @include media($extra-large) {
+    width: $extra-large;
   }
 }
 
-.container--offset {
-  @extend .container;
-  @include media($desktop) {
-    padding-right: 216px;
+.section,
```

Comment:
This would be better:
```
main .container {
  @include section--offset;
}
```
Targeting the content area only and `.section--wide .container` can go alongside `.container`